test/JDBC/expected/Test-sp_helpsrvrolemember-vu-verify.out

```diff
@@ -50,6 +50,31 @@ sysadmin#!#jdbc_user#!#1
 ~~END~~
 
 
```

Comment:
I think you need a truncate table here? Otherwise the SELECT output should have two lines